masterlist.yaml

```
@@ -5992,3 +5992,10 @@ plugins:
         crc: 0x649BC85D
         util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
         itm: 1
+
+- name: 'Magazinemod_OMNI.esp'
+  url: [ 'https://www.nexusmods.com/fallout4/mods/54136' ]
+  inc: [ 'Unofficial Fallout 4 Patch.esp' ]
```

The indentation is off by two spaces for three lines.
```suggestion
  - name: 'Magazinemod_OMNI.esp'
    url: [ 'https://www.nexusmods.com/fallout4/mods/54136' ]
    inc: [ 'Unofficial Fallout 4 Patch.esp' ]
```